### ISSUE TITLE
Time to Trademark the Slogan...? ;-)

### ISSUE THREAD

**Issue by nelsonic** [dwyl/hq#41]
![microsoft-dwyl](https://cloud.githubusercontent.com/assets/194400/11014704/8cf05c16-853a-11e5-97b9-53de43a14a1a.png)

**Comment by iteles** [comment]
:smile: How serious _are_ you?

**Comment by nelsonic** [comment]
It would certainly make headlines if we got the Trademark and sent MSFT a letter... :stuck_out_tongue_winking_eye:

**Comment by tunnckoCore** [comment]
> It would certainly make headlines if we got the Trademark and sent MSFT a letter... 

Hahha yea.. :laughing: sounds good to me.

**Comment by nelsonic** [comment]
@iteles I'm pretty serious... lets re-visit in March when the bank is *refueled* ... :stuck_out_tongue_winking_eye: 
though we *probably* ***don't*** want to get on MSFT radar ... 
@tunnckoCore hope your exams went well!

**Comment by tunnckoCore** [comment]
Yea, almost.. 3 of 6 finished and they weren't so hard, but in anyway I always _trying_ to separate the work and university, lol.

**Comment by markwilliamfirth** [comment]
With regards to the DO WHAT YOU LOVE trade mark

WeWork were [granted ](https://www.ipo.gov.uk/tmcase/Results/23/WE00001294295) the trade mark for this in Dec 2015. It covers classes 35, 36, 41, 42, 45

PERANI & PARTNERS SPA were also [granted ](https://www.ipo.gov.uk/tmcase/Results/4/EU015206584) an image mark with the phrase in 2016

We could file an application for the software class if we decide to do this.

**Comment by nelsonic** [comment]
I'm in no _rush_ to trademark **DO WHAT YOU LOVE** because I don't see us _expanding_ the acronym very often... 
it is _ironic_ that we were using the expression `before` "WeWork" though ... and that this issue was opened `before` they were granted it... 😞

**Comment by iteles** [comment]
Yep. Also assigned to me before the trademark was granted. Too bad I didn't prioritise it 😕

**Comment by markwilliamfirth** [comment]
Sounds like we don't really want or need it! Closed for now 👍

**Comment by markwilliamfirth** [comment]
WeWork registration:

The uk registration number concerned is: WE00001294295
https://www.ipo.gov.uk/tmcase/Results/23/WE00001294295

The international registration number concerned is: 1294295
http://www.wipo.int/madrid/monitor/en/index.jsp

Covers the following:

<img width="845" alt="screen shot 2017-07-28 at 16 48 40" src="https://user-images.githubusercontent.com/5732209/28725191-a4f5171a-73b4-11e7-92e0-9ce5e10680c7.png">

**Comment by markwilliamfirth** [comment]
Posting above as WIPO got back to me by email

This means we could still trademark it for "software" class.

@iteles do we want to reconsider this?

**Comment by iteles** [comment]
@markwilliamfirth Yes :+1: It might not be what we want to be known by but the association is evident and advantageous. 
How much is the UK Trademark?

**Comment by nelsonic** [comment]
dependent on _cost_ we should _consider_ applying for the TM on the Phrase for Software.
It's a _real_ shame that "WeWork" got it in the other categories ... 😞

**Comment by nelsonic** [comment]
dependent on _cost_ we should _consider_ applying for the TM on the Phrase for Software.
(_It's a **real shame** that "WeWork" got it in the other categories ..._ 😞 )

**Comment by markwilliamfirth** [comment]
@iteles @nelsonic 

It costs £170
https://www.gov.uk/how-to-register-a-trade-mark/apply

Do you want to go ahead?
